src/App.js

```diff
@@ -299,60 +299,63 @@ class App extends Component {
   };
 
   render = () => (
-    <MainThemeProvider>
-      <ReduxProvider store={store}>
-        <RainbowContextWrapper>
-          <ErrorBoundary>
-            <Portal>
-              <SafeAreaProvider>
-                <PersistQueryClientProvider
-                  client={queryClient}
-                  persistOptions={persistOptions}
-                >
-                  <RecoilRoot>
-                    <SharedValuesProvider>
-                      <NotificationsHandler
-                        walletReady={this.props.walletReady}
-                      >
-                        <View style={containerStyle}>
-                          {this.state.initialRoute && (
-                            <InitialRouteContext.Provider
-                              value={this.state.initialRoute}
-                            >
-                              <RoutesComponent
-                                onReady={this.handleSentryNavigationIntegration}
-                                ref={this.handleNavigatorRef}
-                              />
-                              <PortalConsumer />
-                            </InitialRouteContext.Provider>
-                          )}
-                          <OfflineToast />
-                          <FedoraToast ref={FedoraToastRef} />
-                        </View>
-                      </NotificationsHandler>
-                    </SharedValuesProvider>
-                  </RecoilRoot>
-                </PersistQueryClientProvider>
-              </SafeAreaProvider>
-            </Portal>
-          </ErrorBoundary>
-        </RainbowContextWrapper>
-      </ReduxProvider>
-    </MainThemeProvider>
+    <Portal>
+      <NotificationsHandler walletReady={this.props.walletReady}>
+        <View style={containerStyle}>
+          {this.state.initialRoute && (
+            <InitialRouteContext.Provider value={this.state.initialRoute}>
+              <RoutesComponent
+                onReady={this.handleSentryNavigationIntegration}
+                ref={this.handleNavigatorRef}
+              />
+              <PortalConsumer />
+            </InitialRouteContext.Provider>
+          )}
+          <OfflineToast />
+          <FedoraToast ref={FedoraToastRef} />
+        </View>
+      </NotificationsHandler>
+    </Portal>
   );
 }
 
-const AppWithRedux = connect(({ appState: { walletReady } }) => ({
-  walletReady,
-}))(App);
+const OldAppWithRedux = connect(state => ({
+  walletReady: state.appState.walletReady,
+}))(OldApp);
 
-const AppWithReduxStore = () => <AppWithRedux store={store} />;
-
-const AppWithSentry = Sentry.wrap(AppWithReduxStore);
+function App() {
+  return <OldAppWithRedux />;
+}
```

Comment:
Added this to this PR after seeing @sammdec [comment here](https://github.com/rainbow-me/rainbow/pull/4209/files#r982615378). I think it makes more sense in this PR, and that PR can then start the migration of `OldApp` into `App` and into other utils or hooks.

Reply:
👍🏻